src/apps/ociocheck/main.cpp

```diff
@@ -209,6 +209,21 @@ int main(int argc, const char **argv)
             for(int i=0; i<config->getNumLooks(); ++i)
             {
                 std::cout << config->getLookNameByIndex(i) << std::endl;
+
+                OCIO::ConstLookRcPtr look = config->getLook(config->getLookNameByIndex(i)); 
+
+                OCIO::ConstTransformRcPtr transform = look->getTransform();         
+
+                try
+                {
+                    OCIO::ConstProcessorRcPtr process = config->getProcessor(transform);
+                }
+                catch(OCIO::Exception & exception)
+                {
+                    std::cout << "\nERROR : " << std::endl;
```

Comment:
I tried this out on a config and it works well, but the formatting is inconsistent with other errors. This is what the output looks like:

```
** Looks **
di

ERROR :
The specified file reference  'look_di.cc' could not be located.
```

Similar ColorSpace errors are formatted like (see lines 186-188):

```
** Looks **
di -- error
    The specified file reference  'look_di.cc' could not be located.
```

General errors, on the other hand, are _mostly_ formatted like (see lines: 267, 270):

```
** Looks **
di
ERROR: The specified file reference  'look_di.cc' could not be located.
```

I'm not overly concerned with which error format is used for looks, but would suggest making it consistent with one of the existing styles for readability.

Reply:
The error messages are clearly inconsistent across the file but also across apps.

The formatting could be:
`std::cerr << "ERROR: " << "Some explanation" << std::endl;`